canonical/vanilla-framework
# Add theme support for table component

## Done

- Update table colour variables to inherit CSS variables

Fixes [WD-8976](https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?selectedIssue=WD-8976)

## QA

- Open [demo](insert-demo-url)
- Check the table examples in the docs, change the parent container to `is-dark` and check if styles update accordingly

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix release (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

[if relevant, include a screenshot or screen capture]


[WD-8976]: https://warthogs.atlassian.net/browse/WD-8976?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
